src/pages/event/WriteEventInfoPage/RecruitForm/RecruitForm.tsx

```diff
@@ -52,8 +58,10 @@ const RecruitForm = () => {
   }, [watch('poster')]);
 
   const onRecruitSubmitForm = (data: FieldValues) => {
-    console.log(data);
-    // TODO: API 연결
+    const eventQueryString = searchParmas[0].get('event') as eventQueryString;
+    const eventType = eventTypeTransform({ eventQueryString });
+    if (isSubmitLoading || !clubId) return;
```

Comment:
p5;
명시적으로 null을 리턴하는건 어떨까요?
isSubmitLoading이 데이터가 도착했는지 판단을 위해서라면
submitForm만 사용해도 될 것 같아요!
```suggestion
    if (submitForm|| !clubId) return;
```

Reply:
아 그 성격 급하신 분들이, 데이터가 보내지지 않았는데 여러번 클릭해서 보냈을 경우에 대비해서,
로딩일 때는 보내지지 않도록 한 코드였습니다!